web/src/main/java/org/openmrs/web/filter/startuperror/StartupErrorFilterModel.java

```diff
@@ -24,6 +24,12 @@ public class StartupErrorFilterModel {
 	
 	public String stacktrace;
 	
+	// automatically given to the .vm files and used there
+	public static final String HEADER_TEMPLATE = "org/openmrs/web/filter/startuperror/header.vm";
+	
+	// automatically given to the .vm files and used there
+	public static final String FOOTER_TEMPLATE = "org/openmrs/web/filter/initialization/footer.vm";
```

Comment:
So we have a common footer for everything? Wouldn't it make sense to move that template up a folder so it isn't part of the rest of the `initialization` files?